libp2p.ss

```diff
@@ -14,7 +14,9 @@
   ;; :vyzo/libp2p/daemon
   current-libp2p-daemon
   start-libp2p-daemon!
+  start-a-libp2p-daemon!
   stop-libp2p-daemon!
+  stop-the-libp2p-daemon!
```

Comment:
Why is it start-a- and stop-the- ?

Reply:
If backwards compatibility is a concern,
Current behavior has:
 - `start-libp2p-daemon!` for "the" daemon in `current-libp2p-daemon`
 - `stop-libp2p-daemon!` for "a" daemon, where `current-libp2p-daemon` doesn't matter

A better design would have:
 - `start-libp2p-daemon!` for "a" daemon
 - `stop-libp2p-daemon!` for "a" daemon
 - `start-the-libp2p-daemon!` for "the" daemon in `current-libp2p-daemon`
 - `stop-the-libp2p-daemon` for "the" daemon in `current-libp2p-daemon`

The closest thing I could think of to that while maintaining backwards compatibility has:
 - `start-a-libp2p-daemon` for "a daemon"
 - `stop-libp2p-daemon!` for "a" daemon as dictated by existing code
 - `start-libp2p-daemon!` for "the" daemon as dictated by existing code
 - `stop-the-libp2p-daemon` for "the" daemon in `current-libp2p-daemon`

If backwards compatibility is not a concern, I could make it more consistent with only "the" and no "a", but I won't do that unless @vyzo says that's what he wants.

Reply:
Okay, I've changed the convention to make it more consistent with only "the" and no "a", and also added some documentation.